tests/components/manual/test_alarm_control_panel.py

```diff
@@ -771,7 +770,7 @@ async def test_trigger_with_pending_and_specific_delay(hass):
 
     state = hass.states.get(entity_id)
     assert state.state == STATE_ALARM_PENDING
-    assert state.attributes["post_pending_state"] == STATE_ALARM_TRIGGERED
+    assert state.attributes["next_state"] == STATE_ALARM_TRIGGERED
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5411564)